modules/ROOT/pages/syntax/expressions.adoc

```diff
@@ -1364,6 +1380,30 @@ RETURN 42 IS :: ANY AS isOfTypeAny, 42 IS :: NOTHING AS isOfTypeNothing
 2+d|Rows: 1
 |===
 
+[[type-predicate-expressions-closed-dynamic-unions]]
+=== Closed Dynamic Unions
+
+_This feature was introduced in Neo4j 5.11._
+
+Closed dynamic union types allow for the testing of multiple types in the same predicate.
+
+This allows for the checking of mixed types:
```

Comment:
I'd just drop this sentence and replace it with a `For example:`. Either in its own paragraph or together with the opening sentence.